src/Servers/HttpSys/src/IHttpSysRequestTimingFeature.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Server.HttpSys;
+
+/// <summary>
+/// This exposes the Http.Sys HTTP_REQUEST_TIMING_INFO.
+/// </summary>
+public interface IHttpSysRequestTimingFeature
+{
+    /// <summary>
+    /// Gets all http.sys timing timestamps
```

Comment:
Nit: @Tratcher how do we normally capitalize "HTTP.sys"? Most of or doc comments seem to go with "Http.sys". We're inconsistent, but we should try to stay consistent within the same document.

Reply:
The namespace is HttpSys, so follow that?

Reply:
Looking at existing public xml comments, it's inconsistent but Http.Sys seems to be used the most. I'll go with that.